mne/report.py

```diff
@@ -814,6 +824,25 @@ def __init__(self, info_fname=None, subjects_dir=None,
 
         self._init_render()  # Initialize the renderer
 
+    def __repr__(self):
+        """Print useful info about report."""
+        s = '<Report | %d items' % len(self.fnames)
+        if self.title is not None:
+            s += ' | %s' % self.title
+        fnames = [_get_fname(f) for f in self.fnames]
+        if len(self.fnames) > 4:
+            s += '\n%s' % '\n'.join(fnames[:2])
+            s += '\n ...\n'
+            s += '\n'.join(fnames[-2:])
+        elif len(self.fnames) > 0:
+            s += '\n%s' % '\n'.join(fnames)
+        s += '\n>'
+        return s
```

Comment:
If you run [this gist](https://gist.github.com/jasmainak/8b2617c7fb4c37f85a363fa9784621b2) and then type in `report` in `ipython`, you will see something like this:

```sh
In [6]: report
Out[6]: 
<Report | 7 items | MNE Report for ...ples/MNE-sample-data
 ... sample_audvis_filt-0-40_raw-eve.fif
 ... sample_audvis_eog-eve.fif
 ...
 ... sample_audvis_raw.fif
 ... Left Auditory
>
```